doc/api/http.markdown

```diff
@@ -43,12 +43,6 @@ list like the following:
       'Host', 'mysite.com',
       'accepT', '*/*' ]
 
-## http.METHODS
-
-* {Array}
-
-A list of the HTTP methods that are supported by the parser.
-
```

Comment:
For reference, here's one of my favorites http://greenbytes.de/tech/webdav/draft-dusseault-caldav-05.html#METHOD_MKCALENDAR CalDAV's MKCALENDAR event which "makes a calendar" and *totally* needs its own HTTP method.